charts/drupal/templates/network-policy.yaml

```diff
@@ -1,6 +1,6 @@
 # Allow access to drupal or varnish pods from the ingress in the silta-cluster namespace
 {{- $id := "allow-ingress" -}}
-{{- $podSelectors := ternary (dict "service" "varnish") (dict "app" "drupal") .Values.varnish.enabled -}}
+{{- $podSelectors := dict "silta-ingress" "allow" -}}
```

Comment:
This needed to be in the drupal chart before because `.Values.varnish.enabled` is not available in the subchart. Now that it's not needed anymore, we can move this whole file to the silta-release chart.

Reply:
Drupal networkpolicy moved to silta-release subchart networkpolicy as a dedicated section.